### PR TITLE
AC Automated Test - LRAC-11364 and LRAC-11365 | master

### PR DESCRIPTION
**Pull request by manuferreira**
https://testray.liferay.com/home/-/testray/case_results?testrayBuildId=1887394858&testrayRunId=1887394883 (CI run)
https://testray.liferay.com/home/-/testray/case_results?testrayBuildId=1889777304&testrayRunId=1889777366 (CI run)
https://issues.liferay.com/browse/LRAC-11364 (automation ticket)
https://issues.liferay.com/browse/LRAC-11365 (automation ticket)

### CODE REVIEW

**Comment by manuferreira** [comment]
ci:test:sf

**Comment by manuferreira** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ACAdmin.testcase [hunk 1]
```diff
@@ -89,4 +92,275 @@ definition {
...
+		task ("Go to AC with Test Test > Settings > User Management > Invite the new user to the workspace") {
+			ACUtils.launchAC();
+
+			ACNavigation.goToSettings();
+
+			ACSettings.goToUserManagement();
+
+			ACSettings.addUser(userInvite = "ac@liferay.com");
+
+			ACUtils.searchBar(searchItem = "ac@liferay.com");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ACAdmin.testcase:290]: Hey @manuferreira, I think this search here is not necessary. What do you think?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ACAdmin.testcase [hunk 2]
```diff
@@ -89,4 +92,275 @@ definition {
...
+				locator1 = "TextInput#EMAIL_ADDRESS",
+				value1 = "ac@liferay.com");
+
+			Type(
+				locator1 = "TextInput#PASSWORD",
+				value1 = "test");
+
+			Click(
+				locator1 = "Button#SIGN_IN",
+				value1 = "Sign In");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ACAdmin.testcase:174]: Hey @manuferreira, I think you could log in using this macro. What do you think?

```
ACLogin.loginAs(
				emailAddress = "ac@liferay.com",
				password = "test");
```

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ACAdmin.testcase [hunk 3]
```diff
@@ -89,4 +92,275 @@ definition {
...
+		}
+
+		task ("Assert that the user does not have permission to access the admin page") {
+			AssertTextEquals(
+				locator1 = "Portlet#ERROR",
+				value1 = "You do not have the roles required to access this portlet.");
+		}
+	}
+
+	@description = "Bug: LRAC-11339 | Automation ID: LRAC-11365 | Test Summary: A new user created in the admin part can be invited to the workspace and see the data, the same user now with admin permission also can see the data"
```
Comment (joshitagaki) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ACAdmin.testcase:198]: Can you change `part` to `page`?

> Test Summary: A new user created in the admin `page`

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ACAdmin.testcase (hunk 2 above)
Comment (manuferreira) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ACAdmin.testcase:174]: I will check it out. Thanks :)

**Comment by manuferreira** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ACAdmin.testcase [hunk 4]
```diff
@@ -89,4 +92,265 @@ definition {
...
+
+			Type(
+				locator1 = "UsersAndOrganizationsEditUser#PASSWORD_NEW_PASSWORD_FIELD",
+				value1 = "test");
+
+			Type(
+				locator1 = "UsersAndOrganizationsEditUser#PASSWORD_ENTER_AGAIN_FIELD",
+				value1 = "test");
+
+			Button.clickSave();
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ACAdmin.testcase:154]: Hey @manuferreira, couldn't the macros we use to put passwords on users in DXP work here? As this part of /admin is a DXP 7.2, I think you could try it, I'll leave two suggestions for you and I think if any works, maybe there's a way to create the users with the DXP macros as well:

- User.editPasswordCP
- JSONUser.setFirstPassword

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ACAdmin.testcase [hunk 5]
```diff
@@ -89,4 +92,265 @@ definition {
...
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ACAdmin.testcase:270]: Hey @manuferreira, couldn't the macros we use to put passwords on users in DXP work here? As this part of /admin is a DXP 7.2, I think you could try it, I'll leave two suggestions for you and I think if any works, maybe there's a way to create the users with the DXP macros as well:

- User.editPasswordCP
- JSONUser.setFirstPassword

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ACAdmin.testcase (hunk 4 above)
Comment (manuferreira) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ACAdmin.testcase:154]: Hey Vini, I tested using a few macros and all of them failed at some point:
- User.editPassword
- User.editPasswordCP
- JSONUser.setFirstPassword
I also tried the JSON macro to create a user, but the user is only created on DXP and not where I want.

**Review by joshitagaki — APPROVED** [review]
LGTM! Thanks @manuferreira!

**Comment by manuferreira** [comment]
ci:test:sf

**Comment by manuferreira** [comment]
Hi, @christianstokes mentioned a function `User.addCP` but unfortunately this function failed later in the test but this led me to test other functions inside `User.macro` file and some of them worked (`User.resetPassword` and `User.assignRegularRoleCP`), so I erased some commits that would no longer be needed. 
cc: @joshitagaki @ViniLopes20

**Comment by ViniLopes20** [comment]
ci:forward

**Comment by ViniLopes20** [comment]
ci:forward

**Pull request closed without merging**